### DISCUSSION TITLE
Predicted low alert requires phone on "loud"?

### DISCUSSION THREAD

**Discussion by corneliusroemer** [NightscoutFoundation/xDrip#2034] [Q&A]
I find the predicted low alert the most useful alert.

Unfortunately, it neither vibrates nor makes a sound unless the phone is in "Ring" mode, which is almost never the case for me.

So the only way I see it is in Notifications.

Would it be possible to add a vibration pattern to it? And maybe allow overwriting the silent/vibrate setting?

How do other people here handle this?

I find alarm options in xdrip+ quite confusing. Otherwise the app is of course amazing, so much better than Dexcom!

I'm talking about this alert in the middle:

![Screenshot_20220305-153327.jpg](https://user-images.githubusercontent.com/25161793/156888075-2f8c5079-361a-4436-8968-e270e599b9e1.jpg)

**Comment by Navid200** [comment]
The alerts are not consistent because they have been created by different people over a long period of time.
You can see my dream to make them consistent here:
https://github.com/NightscoutFoundation/xDrip/issues/1762

However, some of the differences are intentional.
For example, if you have a low alert, as long as it continues to be low, and as long as you have not snoozed it, it is supposed to keep going at the specified interval.
That would make no sense for predicted low.  
The fast rising or fast falling alerts are intentionally meant not to be intrusive.  They are meant to be a notification rather than an alarm.

I am not a programmer.  It takes me a very long time to complete a submission.  What is on my list right now is to add a choice of sound to the missed signal alert.
Someone else may beat me to it.

**Comment by Navid200** [comment]
I don't know what you mean when you say it makes no noise unless it is in ring mode.

You need to raise the ring and notifications volume.  If you lower that volume to 0, it will make no noise.  Can you not raise that volume?
